components/espcoredump/include_core_dump/esp_core_dump_types.h

```diff
@@ -66,8 +66,8 @@ extern "C" {
                                                 (((_maj_)&0xFF) << 8) | \
                                                 (((_min_)&0xFF) << 0) \
                                             )
-#define COREDUMP_VERSION_BIN                0
-#define COREDUMP_VERSION_ELF                1
+#define COREDUMP_VERSION_BIN                1
+#define COREDUMP_VERSION_ELF                2
```

Comment:
That's not correct, unfortunately — this changes the definition of COREDUMP_VERSION_BIN_LEGACY a few lines below, meaning that the coredumps in legacy format won't be encoded correctly, and older versions of esp-coredump won't be able to read them.